static/js/containers/CourseSearchPage_test.js

```diff
@@ -171,8 +175,9 @@ describe("CourseSearchPage", () => {
         })
       )
     })
+    wrapper.update()
     // from is 5, plus 5 is 10 which is == numHits so no more results
-    assert.isFalse(inner.find("InfiniteScroll").prop("hasMore"))
+    assert.isFalse(wrapper.find("InfiniteScroll").prop("hasMore"))
```

Comment:
Should this `"InfiniteScroll"` be unquoted? and below as well?

Reply:
doesn't seem to matter either way, so I think I'll make them all quoted for consistency